fix-point/src/Refactor.hs

```diff
@@ -0,0 +1,29 @@
+module Rename where 
+
+changeLine :: String -> String -> String 
+changeLine newMod line = case words line of 
+    -- module names 
+    "module" : s : extra     -> unwords ("module": r1 : extra)
+    -- imports which start with Lib
+    "import" : s : extra     -> if take (length "Lib") s == "Lib" 
+                                then unwords ("import": r1 : extra) 
+                                else line 
+    -- reexport where reexport module started with Lib
+    "(":"module" : s : extra -> unwords ("(" :"module": r2 : extra)
+    ",":"module" : s : extra -> unwords ("," :"module": r2 : extra)
+    -- Anything else can't be changed
+    _ -> line
+  where
+    r1 = renamePrefix newMod $ splitAt (length "Lib") (words line !! 1)
```

Comment:
It's better to move `length "Lib"` into single variable called `libLen` to avoid multiple calls to `length` function.